source/index.txt

```diff
@@ -25,28 +25,28 @@ MongoDB Realm Application Services
    Values & Secrets </values-and-secrets>
    Logs </logs>
    Activity Feed & Alerts </alerts>
-   Realm CLI </cli>
+   {+cli+} </cli>
    Reference </reference>
    Release Notes </release-notes>
-   Realm SDKs <https://mongodb.com/docs/realm>
+   {+client-sdks+} <https://mongodb.com/docs/realm>
    Get Help </help>
 
-MongoDB Realm provides a suite of managed cloud services including Realm Sync,
+{+service+} provides a suite of managed cloud services including {+sync+},
 serverless cloud functions, declarative data access rules, and more. You can use
-Realm's services to write and host a full application in a managed cloud
+{+service-short+}'s services to write and host a full application in a managed cloud
 environment backed by MongoDB Atlas.
 
 .. card-group::
    :style: extra-compact
    :columns: 2
 
    .. card::
-      :headline: Create a New Realm App
+      :headline: Create a New {+app+}
       :url: https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui/
       :icon: /images/icons/realm/cross-platform.svg
-      :icon-alt: Realm App Icon
+      :icon-alt: {+app+} Icon
       
-      Realm Apps contain all of your hosted services. Learn how to set up your
+      {+apps+} contain all of your hosted services. Learn how to set up your
```

Comment:
Looks like there's no plural "apps" source constant. Can we add one?

Reply:
Or maybe there is, but there's something wrong with this card. Take a look and see if you can figure it out.

Reply:
Honestly I'm thinking of going back through now and replacing all the source constants with the correct text anyway...